roles/system-metrics/templates/kube-burner.yml.j2

```diff
@@ -0,0 +1,42 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: system-metrics-collector-{{ trunc_uuid }}
+  namespace: {{ operator_namespace }}
+spec:
+  backoffLimit: 0
+  activeDeadlineSeconds: 3600
+  template:
+    metadata:
+      labels:
+        app: system-metrics-collector-{{ trunc_uuid }}
+    spec:
+{% if workload_args.runtime_class is defined %}
+      runtimeClassName: "{{ workload_args.runtime_class }}"
+{% endif %}
+      restartPolicy: Never
+      containers:
+      - name: kube-burner
+        image: quay.io/cloud-bulldozer/kube-burner:latest
```

Comment:
Thoughts on adding the optional image argument here as well?

Reply:
done